util/admin/admin.go

```diff
@@ -91,6 +91,12 @@ func isJobRollbackable(job *model.Job, id int64) error {
 			job.SchemaState == model.StateDeleteReorganization {
 			return ErrCannotCancelDDLJob.GenWithStackByArgs(id)
 		}
+	case model.ActionDropColumn:
+		if job.SchemaState == model.StatePublic || job.SchemaState == model.StateDeleteOnly ||
```

Comment:
I think check can be simply to:
```go
if job.SchemaState != model.StateNone {
	return error .....
}
```
model.StateNone mean the job still does not run.